plugins/legion-go2/meson.build

```diff
@@ -0,0 +1,23 @@
+# plugins/legion-go2/meson.build
+# SPDX-License-Identifier: LGPL-2.1-or-later
+# Copyright 2025 lazro
```

Comment:
Copyright has to go first, license second. Copyright should include a company name or an email address.